pkg/cvo/cvo_scenarios_test.go

```diff
@@ -1974,9 +1975,9 @@ func TestCVO_UpgradeFailedPayloadLoadWithCapsChanges(t *testing.T) {
 				{Type: ImplicitlyEnabledCapabilities, Status: "False", Reason: "AsExpected", Message: "Capabilities match configured spec"},
 				{Type: DesiredReleaseAccepted, Status: configv1.ConditionFalse, Reason: "VerifyPayloadVersion",
 					Message: "Verifying payload failed version=\"1.0.0-abc\" image=\"image/image:0\" failure=release image version 1.0.1-abc does not match the expected upstream version 1.0.0-abc"},
-				{Type: "Available", Status: "True", Message: "Done applying 1.0.1-abc"},
+				{Type: "Available", Status: "False"},
```

Comment:
Do we understand why we're `Available=False` here with no `Reason` or `Message`?

Reply:
This final status reaping has always been very time sensitive. Appears with the lock changes allowing the Update more time the sync doesn't get as far, and does not complete. To get to completion I reduced test payload to one file and put in an additional wait for the completion of sync'ing the original payload.